fontgenerator.js

```diff
@@ -435,6 +433,11 @@ TinyFontGenerator = {
     data += " 00 00 00 00";  // italicAngle
     data += " 00 00 00 00";  // underlinePosition
     data += " 00 00 00 00";  // underlineThickness
+    data += " 00 00 00 00";  // isFixedPitch
```

Comment:
well, at least `isFixedPitch` is specified in https://github.com/khaledhosny/ots/blob/master/src/post.h. I'm not sure why it isn't raising an error at https://github.com/khaledhosny/ots/blob/master/src/post.cc#L26